abel/benchmark.py

```diff
@@ -104,38 +104,66 @@ def __init__(self, n=[301, 501], select=["all", ], n_max_bs=700,
         # ---- timing tests for various image sizes nxn
         for ni in n:
             ni = int(ni)
-            x = np.random.randn(ni, ni)
-           
+            h, w = ni, ni//2 + 1
```

Comment:
This is good, a 1/2 sized odd-image is even, but the correct column size is not printed in the benchmark output
```
Implementation      n = 501       
```

Reply:
So, for n=501, we are passing `three_point` (for example) a `h=501, w=251` half-image. This is what most users would employ if they had a `501x501` image from their CCD. So, I'm calling this the "501x501" image. Does that seem reasonable?

Reply:
Only, that it is not the image size that is timed. Perhaps, I am being too pedantic.